docs/Tutorials/Advanced/ticketing/user-code.md

```diff
@@ -0,0 +1,158 @@
+---
+sidebar_label: 'Claim Page Code'
+---
+# Claim Page Code
+
+## Introduction
+In the previous section, you broke down the flow and logic of the ticket app. You also learned what Keypom information is needed in order to render the pages properly. 
+
+In this tutorial, you'll learn about the actual code needed to create the claim page for your seamless ticketing system.
+
+:::note
+The focus of this section will be on the Keypom aspects of each script, such as getting key information and using it. 
+
+The full code for each script will be shown, with highlights. For React resources, see [here](https://reactjs.org/docs/hello-world.html) as a starting point.
+:::
+
+Recall, from the [introduction](introduction.md) that your code had the following outline.
+
+```bash
+/my-ticket-app
+├── contract
+│   └── ...
+├── frontend
+│   └── components
+│   │    └── scanner.js
+│   │    └── qrcode.js
+│   └── state
+│   │    └── App.js
+│   │    └── keyInfo.js
+│   └── utils
+│   │    └── configurations.js
+│   │    └── createTickDrop.js
+│   └── node_modules
+│   │    └── keypom-js
+│   │    └── qrcode.react
+│   │    └── react-zxing
+│   │    └── react
+│   │    └── react-dom
+│   │    └── react-router-dom
+│   │    └── ...
+│   └── package.json
+│   └── package-lock.json
+├── ...
+├── package.json
+├── package-lock.json
+```
+
+This tutorial will be covering the code in `app.js`, `KeyInfo.js`, and `qrcode.js`.
+
+---
+
+## `App.js`
+### Setting Up
+The primary purpose of `app.js` is to display the different states of the claim page. This will involve getting the current key uses, and then rendering based on the value returned. 
+
+The first step is to initialize a connection to NEAR and setup all the state variables that will be needed to render the page. 
+
+```jsx reference showLineNumbers
```

Comment:
This code snippet is cutoff
<img width="649" alt="image" src="https://user-images.githubusercontent.com/57506486/224809788-53b4d5ca-5eeb-4196-b994-237ef110fd54.png">